### ISSUE TITLE
Demo missing file

### ISSUE THREAD

**Issue by HarveyDawn** [algolia/voice-overlay-ios#11]
error: missing module map file: '/Users/Harvey/Downloads/voice-overlay-ios-master/VoiceOverlay-Demo/Pods/Target Support Files/InstantSearchVoiceOverlay/InstantSearchVoiceOverlay.modulemap (in target 'InstantSearchVoiceOverlay')

**Comment by spinach** [comment]
Hey @Harveyhw1991 - Can you run `pod update` and see if that solves the issue ?

**Comment by spinach** [comment]
Feel free to re-open the issue if you still can't run the demo project
